buildspecs/j9.flags

```diff
@@ -1832,7 +1832,7 @@ Only available on zOS</description>
 		<description>Enables support for Foreign Linker API (part of Project Panama) such as native invocations to ffi_call</description>
 		<ifRemoved>No support for Foreign Linker API</ifRemoved>
 	</flag>
-	<flag id="opt_ojdkThreadSupport">
+	<flag id="opt_openjdkThreadSupport">
 		<description>Enables OpenJDK Thread support for Project Loom</description>
 		<ifRemoved>No OpenDJK Thread support for Project Loom</ifRemoved>
```

Comment:
Since we are here, please help fix this typo, it should be `OpenJDK`.

Reply:
I should have noticed and fixed that without your prompt: fixed.